src/commands/save-cache.yml

```diff
@@ -0,0 +1,11 @@
+description: "Save Composer deps to cache."
+parameters:
+  key:
+    description: "The cache key to use. The key is immutable."
+    type: string
+    default: "composer"
+steps:
+  - save_cache:
+      key: << parameters.key >>-{{ checksum "composer.lock"  }}
+      paths:
+        - "vendor"
```

Comment:
Complaining of no new line at end of file. This will fail linting.

Reply:
resolved in latest commit